irohad/main/server_runner.cpp

```diff
@@ -28,8 +35,37 @@ iroha::expected::Result<int, std::string> ServerRunner::run() {
     builder.AddChannelArgument(GRPC_ARG_ALLOW_REUSEPORT, 0);
   }
 
-  builder.AddListeningPort(
-      serverAddress_, grpc::InsecureServerCredentials(), &selected_port);
+  bool enable_tls = tlsKeypair_.length() > 0;
+  if (enable_tls) {
```

Comment:
Probably the whole TLS initialization code can be moved to a separate method.
Please consider that option. Otherwise, ``run`` method comes quite big.

Reply:
Fixed in latest version